pages/model/[uuid].tsx

```diff
@@ -168,6 +179,25 @@ function Model() {
     await postEndpoint(`/api/v1/version/${version?._id}/reset-approvals`, {}).then((res) => res.json())
   }
 
+  const handleDelete = () => {
+    setDeleteConfirmContent(<Box />)
+    setDeleteConfirmOpen(true)
+  }
+
+  const handleDeleteCancel = () => {
+    setDeleteConfirmOpen(false)
+  }
+
+  const handleDeleteConfirm = async () => {
+    await deleteEndpoint(`/api/v1/model/${uuid}`, {}).then((res) => {
+      if (res.status < 400) {
```

Comment:
Should we not be checking that `res.status >= 200 && res.status < 300`? Also, as this is the sort of logic that we'll want to reuse, is it worth creating a reusable util function for this?